internal/commands/secrets/output.go

```diff
@@ -1,6 +1,56 @@
 package secrets
 
+import (
+	"github.com/10gen/realm-cli/internal/cloud/realm"
+	"github.com/10gen/realm-cli/internal/terminal"
+)
+
 const (
-	headerID   = "ID"
-	headerName = "Name"
+	headerID      = "ID"
+	headerName    = "Name"
+	headerDeleted = "Deleted"
+	headerDetails = "Details"
+
+	secretDeleteMessage = "Deleted Secrets"
 )
+
+type secretOutputs []secretOutput
+
+type secretOutput struct {
+	secret realm.Secret
+	err    error
+}
+
+type secretTableHeaderModifier func() []string
+type secretTableRowModifier func(secretOutput, map[string]interface{})
+
+func secretOutputComparerBySuccess(outputs secretOutputs) func(i, j int) bool {
+	return func(i, j int) bool {
+		return outputs[i].err != nil && outputs[j].err == nil
+	}
+}
+
+func secretHeaders(modifier secretTableHeaderModifier) []string {
```

Comment:
[question] could this just be `func secretHeaders(additionalHeaders ...string) []string`?